picard/file.py

```diff
@@ -140,11 +140,14 @@ def copy_metadata(self, metadata):
             values = self.orig_metadata.getall(tag)
             if values:
                 saved_metadata[tag] = values
+        deleted_tags = set(self.metadata.deleted_tags)
```

Comment:
Isn't `self.metadata.deleted_tags` already a *set* object. Although it doesn't affect the `deleted_tags` object as *set* of an existing *set* object is just a *set* (which I just came to know :D), I was wondering if this was intentional.